dashboard/app/models/script.rb

```diff
@@ -180,7 +180,16 @@ def self.script_cache_from_cache
   def self.script_cache_from_db
     {}.tap do |cache|
       Script.all.pluck(:id).each do |script_id|
-        script = Script.includes([{script_levels: [{levels: [:game, :concepts] }, :stage, :callouts]}, :stages]).find(script_id)
+        script = Script.includes([
+          {
+            script_levels: [
+              {levels: [:game, :concepts, :level_concept_difficulty]},
```

Comment:
Note that we are caching `LevelConceptDifficulty` via `Level` rather than in isolation. Criticism welcome.